### ISSUE TITLE
"Protected Zone" NPC Theft Prevention

### ISSUE THREAD

**Issue by Solusphere** [CleverRaven/Cataclysm-DDA#27697]
Currently, any NPC can casually walk up to you, offer you a job, and then pick up all of your meds and cash cards and walk off with them, requiring you to either kill them/engineer a mysterious accident or sit there and rage impotently. There are ways you can deal with this, such as by preventing line of sight, but nothing that's particularly convenient.

I propose a system where you mark an area as a Protected Zone. This would essentially be something akin to a sign saying "Touch my stuff and I'll **** your **** in the ***** with my *****." In it's simplest incarnation, this would just require an NPC to check their current fear of you (which is already something the game tracks from what I understand) before attempting to take any items from the designated space. If they fear you enough, they won't take anything. If they don't fear you enough, and decide to take your stuff anyway, then it should remove the "Killed an Innocent" morale penalty for killing them, or at least cause a less severe version of the debuff. Even if this only works while you're there and watching them, it would be an improvement over the current options.

Additional features which could potentially be added to this, but which would require more work:

- Allow NPC guards to also influence the check, so you can have them guard your item stashes. Not sure what would happen if the thieving NPC steals something anyway. The guard becomes hostile and attacks them maybe?
- Allow certain fear-inducing items to increase your fear rating, such as effigies, severed heads, turrets, leashed dogs etc.

**Comment by kevingranade** [comment]
Have some history:
https://github.com/CleverRaven/Cataclysm-DDA/issues/9555
https://discourse.cataclysmdda.org/t/claim-system/7579/13
Tl;dr this is very similar to what I had in mind, but you added some neat twists.

**Comment by aoae23** [comment]
What exactly is meant by "effigies"?

**Comment by Solusphere** [comment]
> What exactly is meant by "effigies"?

The actual definition is just a sculpture of a person, but it's usually used to to mean a representation of something/someone being burned, tortured etc, to cause fear or express hatred. So for example, it could be a human skeleton nailed to a cross, or other such grisly totem. It would be mostly intended for the more psychopathic, cannibal-type players. See the game The Forest for many examples.

**Comment by Treah** [comment]
Maybe we can add in a lockable box or something that you can craft to keep items that are valuable. Possibly require you to find a padlock with its keys and its used in the crafting menu. In this case it would be a construction but have the same properties that vehicle storage does where a NPC cannot remove items.

**Comment by mlangsdorf** [comment]
See also #27467, which proposes a more general system of ownership that would also limit the ability of PCs to steal from friendly but non-allied NPCs.
